### PR TITLE
prepare website for GRASS 8 release

### PR DESCRIPTION
**Pull request by veroandreo**
I've started preparing the website for upcoming GRASS 8 release. I tried to homogenize terminology and version numbering among different subsections under Download. So far, I left 8.0.0 (current), 7.8.6 (legacy) and 8.2.0 (preview). I remember there was a discussion about terminology but couldn't find it. Please have a look/comment and update links where needed (for windows, I have left the RC link in current and the ctu link for dev).

Do you agree with using 3 digits for versions and the names current, legacy and preview? Should we use devel(olpment) instead of preview?

Also, I commented out the startup screen pic in Download/data as it's gone in 8. What would be a good replacement for that functionality?

Next, I'll add a news item as well. What else do we need to change?

### CODE REVIEW

File: content/download/linux.en.md [hunk 1]
```diff
@@ -25,40 +25,44 @@ list of GRASS GIS packages.
...
 
 <div class="alert rounded-0 alert-success">
-<i class="fa fa-info-circle"></i> <u>Current stable release</u>, see <a href="https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures78">GRASS GIS 7.8 new features</a> and <a href="https://trac.osgeo.org/grass/wiki/Release/7.8.6-News">GRASS GIS 7.8.6 announcement</a> for more information.</div>
+<i class="fa fa-info-circle"></i> <u>Current stable release</u>, see <a href="https://trac.osgeo.org/grass/wiki/Grass8/NewFeatures80">GRASS GIS 8.0 new features</a> and <a href="https://trac.osgeo.org/grass/wiki/Release/8.0.0-News">GRASS GIS 8.0.0 announcement</a> for more information.</div>
 
-*  [Generic 64bit](/grass78/binary/linux/snapshot) (weekly binary snapshot)
+*  [Generic 64bit](/grass80/binary/linux/snapshot) (weekly binary snapshot)
 *  [Gentoo](https://packages.gentoo.org/packages/sci-geosciences/grass)
-*  [EPEL7](https://copr.fedorainfracloud.org/coprs/neteler/grass78/) (RHEL7/Centos7/Scientific Linux7)
+*  [EPEL7](https://copr.fedorainfracloud.org/coprs/neteler/grass80/) (RHEL7/Centos7/Scientific Linux7)
```
Comment (neteler) [inline content/download/linux.en.md:35]:
```suggestion
```

EPEL7 binaries do not exists due to lack of dependencies in EPEL7, see https://copr.fedorainfracloud.org/coprs/neteler/grass80/build/3189058/

File: content/download/linux.en.md [hunk 2]
```diff
@@ -25,40 +25,44 @@ list of GRASS GIS packages.
...
-<i class="fa fa-info-circle"></i> <u>Old stable release</u>, see <a href="https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures76">GRASS GIS 7.6 new features</a> and  <a href="https://trac.osgeo.org/grass/wiki/Release/7.6.1-News">GRASS GIS 7.6.1 announcement</a> for more information.
+<i class="fa fa-info-circle"></i> <u>Old stable release</u>, see <a href="https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures78">GRASS GIS 7.8 new features</a> and  <a href="https://trac.osgeo.org/grass/wiki/Release/7.8.6-News">GRASS GIS 7.8.6 announcement</a> for more information.
 </div>
 
-*  [Generic 64bit](/grass76/binary/linux/snapshot) (weekly binary snapshot)
-*  [EPEL7](https://copr.fedorainfracloud.org/coprs/neteler/grass76/) (RHEL7/Centos7/Scientific Linux7)
-*  [Fedora](https://copr.fedorainfracloud.org/coprs/neteler/grass76/)
+*  [Generic 64bit](/grass78/binary/linux/snapshot) (weekly binary snapshot)
+*  [Gentoo](https://packages.gentoo.org/packages/sci-geosciences/grass)
+*  [EPEL7](https://copr.fedorainfracloud.org/coprs/neteler/grass78/) (RHEL7/Centos7/Scientific Linux7)
```
Comment (neteler) [inline content/download/linux.en.md:51]:
```suggestion
```
EPEL7 binaries do not exist due to lack of dependencies in EPEL7, see https://copr.fedorainfracloud.org/coprs/neteler/grass78/build/3218897/

File: content/download/docker.en.md [hunk 3]
```diff
@@ -19,51 +19,55 @@ For a version matrix (GRASS GIS, PROJ, GDAL, PDAL), see [here](https://github.co
...
-### <a name="GRASS-GIS-devel"></a> GRASS GIS 8.0 (preview)
+### <a name="GRASS-GIS-devel"></a> GRASS GIS 8.2.0 (preview)
 
 <div class="alert rounded-0 alert-info">
-<i class="fa fa-info-circle"></i> The following Docker images are generated from the <b>GRASS GIS master branch</b>
+<i class="fa fa-info-circle"></i> The following Docker images are generated from the <b>GRASS GIS main branch</b>
 </div>
 
-*  [<i class="fa fa-download"></i> GRASS GIS 8.0 development, Ubuntu 18.04 based](https://hub.docker.com/r/neteler/grassgis7/) (<small>git main, grass80, 520 MB, with Python 3</small>)
+*  [<i class="fa fa-download"></i> GRASS GIS 8.2 development, Ubuntu 18.04 based](https://hub.docker.com/r/neteler/grassgis7/) (<small>git main, grass80, 520 MB, with Python 3</small>)
```
Comment (neteler) [inline content/download/docker.en.md:63]: This is outdated since dockerhub changed their policy to paid accounts.
Hence the entry may be removed.

**Comment by veroandreo** [comment]
I have now added the first version of the news. I have recycled/adapted the content from the `new features` page:

1)
image
2)
image
3)
image
4)
image

The news file date is Jan 30 :wink: :wink:

File: content/news/2022_01_30_GRASS_GIS_8_0_0_released.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+Major efforts have been made to port all Python scripts to Python 3.6.
+Also, there's a new Python subpackage for working with locations and mapsets.
+All Python code has been formatted with `black` and checked against `flake8`.
+Most warnings in C code were removed and are now checked and, all code 
+is tested with CodeQL.
+
+#### PROJ and GDAL support
+
+PROJ 4 to PROJ 7 compatibility has been implemented. WKT2 can be
```
Comment (neteler) [inline content/news/2022_01_30_GRASS_GIS_8_0_0_released.md:130]:
```suggestion
PROJ 4 to PROJ 8 compatibility has been implemented. WKT2 can be 
```

(on Fedora 35 I work with PROJ 8.2.1, January 1st, 2022)

File: content/news/2022_01_30_GRASS_GIS_8_0_0_released.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+#### Binaries/Installers download:
+
+- winGRASS 8.0.0/standalone:
+  [64bit standalone installer](/grass80/binary/mswindows/native/WinGRASS-8.0.0-1-Setup.exe)
+- winGRASS 8.0.0/OSGeo4W:
+  [64bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/v2/osgeo4w-setup.exe)
+- Mac
+    - [Official Application Bundle](http://grassmac.wikidot.com/downloads)
+    - ~~ [MacPorts](https://ports.macports.org/port/grass7/) ~~
```
Comment (nilason) [inline content/news/2022_01_30_GRASS_GIS_8_0_0_released.md:152]:
```suggestion
    -  [MacPorts](https://ports.macports.org/port/grass/)
```

I plan to suggest to revive the un-versioned `grass` port for GRASS v8 (it has been a, now obsolete, remnant of the v6 era).

File: content/news/2022_01_30_GRASS_GIS_8_0_0_released.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+#### PROJ and GDAL support
+
+PROJ 4 to PROJ 7 compatibility has been implemented. WKT2 can be 
+used for CRS definitions. High-accuracy coordinate transformations supported. 
+GDAL 3 compatibility improved with speed-ups.
+
+#### Full list of changes and new features
+
+The overview of new features in the 8.0 release series is available at
+[new features in GRASS GIS 8.0](https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures80).
```
Comment (nilason) [inline content/news/2022_01_30_GRASS_GIS_8_0_0_released.md:137]:
```suggestion
[new features in GRASS GIS 8.0](https://trac.osgeo.org/grass/wiki/Grass8/NewFeatures80).
```

File: content/news/2022_01_30_GRASS_GIS_8_0_0_released.md (hunk 4 above)
Comment (nilason) [reply content/news/2022_01_30_GRASS_GIS_8_0_0_released.md:130]: > (on Fedora 35 I work with PROJ 8.2.1, January 1st, 2022)

Same for me with MacPorts.

**Comment by neteler** [comment]
News: GRASS GIS 8.0.0 has been released :partying_face: 

https://github.com/OSGeo/grass/releases/tag/8.0.0

**Comment by veroandreo** [comment]
I'll wait for windows packages to be ready for merging, is that ok? Otherwise, Planet OSGeo will automatically retweet. 
Besides, there's https://github.com/OSGeo/grass/issues/2137... Maybe we need to change the announcement??

**Comment by nilason** [comment]
> I'll wait for windows packages to be ready for merging, is that ok? Otherwise, Planet OSGeo will automatically retweet.
> Besides, there's [OSGeo/grass#2137](https://github.com/OSGeo/grass/issues/2137)... Maybe we need to change the announcement??

I'm sad to say it would be really bad to announce this now. Compilation with a repo (eg. official tarball) without .git, is totally broken. It's a no no, in my opinion. Bad enough that no-one of us discovered this till now.

**Comment by veroandreo** [comment]
I now also replaced the image to download sample data from GUI 
image

File: content/download/windows.en.md [hunk 7]
```diff
@@ -21,35 +21,33 @@ layout: "os"
...
-#### <a name="GRASS-GIS-new-stable"></a> GRASS GIS 7.8.6 (current)
+#### <a name="GRASS-GIS-current"></a> GRASS GIS 8.0.0 (current)
 
 <div class="alert rounded-0 alert-success">
-<i class="fa fa-info-circle"></i> <u>New stable release</u>, see <a href="https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures78">GRASS GIS 7.8 new features</a> and <a href="https://trac.osgeo.org/grass/wiki/Release/7.8.6-News">GRASS GIS 7.8.6 announcement</a> for more information.</div>
+<i class="fa fa-info-circle"></i> <u>New stable release</u>, see <a href="https://trac.osgeo.org/grass/wiki/Grass8/NewFeatures80">GRASS GIS 8.0 new features</a> and <a href="https://trac.osgeo.org/grass/wiki/Release/8.0.0-News">GRASS GIS 8.0.0 announcement</a> for more information.</div>
 
-*  [<i class="fa fa-download"></i> Download 64bit](/grass78/binary/mswindows/native/x86_64/WinGRASS-7.8.6-1-Setup-x86_64.exe)
-*  [<i class="fa fa-download"></i> Download 32bit](/grass78/binary/mswindows/native/x86/WinGRASS-7.8.6-1-Setup-x86.exe)
+*  [<i class="fa fa-download"></i> Download 64bit](/grass80/binary/mswindows/native/WinGRASS-8.0.0RC2-1-Setup.exe)
```
Comment (landam) [inline content/download/windows.en.md:29]:
```suggestion
- *  [<i class="fa fa-download"></i> Download 64bit](/grass80/binary/mswindows/native/WinGRASS-8.0.0RC2-1-Setup.exe)
+ *  [<i class="fa fa-download"></i> Download 64bit](/grass80/binary/mswindows/native/WinGRASS-8.0.0-1-Setup.exe)
```

**Pull request merged**